frontend/app/templates/components/pricing-component.hbs

```diff
@@ -6,43 +6,60 @@
         <p class="heading-alpha">Pricing</p>
         <br>
         <div class="inline-columns">
-            <div class="ui stackable two column grid container center aligned">
+            <div class="ui stackable three column grid container center aligned">
                 <div class="column">
                     <div class="pdf icon-pdf">
                         <div class="pdf-div">
                             <img class="pdf-img" src="/images/pdf-icon.png">
                         </div>
-                        <div class="heading-bravo">You print the generated PDF</div>
+                        <div class="heading-bravo">Less than 200 badges</div>
+                        <div class="cb-price">
+                            <span class="cb-price-number">Free</span>
+                        </div>
+                        <div class="ui list">
+                            <div class="item">All features are identical with the paid options</div>
+                            <div class="item">Only 200 badges can be generated in total</div>
+                            <div class="item">The badges shall be print on your own</div>
+                        </div>
+                    </div>
+                </div>
+                <div class="column">
+                    <div class="border-left"></div>
+                    <div class="shipped icon-box">
+                        <div class="grp-div">
+                            <img class="grp-img" src="/images/blog.png">
+                        </div>
+                        <div class="heading-bravo">More than 200 badges</div>
                         <div class="cb-price">
                             <span class="cb-price-tag">$</span>
-                            <span class="cb-price-number">0.00</span>
-                            <span>for </span>
+                            <span class="cb-price-number">1.00</span>
```

Comment:
The price rights should be given to admin , Create a new field if its not already there

Reply:
Price rights are for using badgeyay.